ingest/ingest.py

```diff
@@ -1,359 +0,0 @@
-    result['created_at'] = repo_json['created_at']
-    result['language'] = repo_json['language']
-    result['metrics'] = repo_json['metrics']
-    result['organization'] = repo_json['org_obj']
-    result['origin'] = 'PUBLIC'
-    result['project_name'] = repo_json['project_name']
-    result['stage_id'] = repo_json['_id']
-    result['stage_source'] = "SONAR"
-    result['updated_at'] = repo_json['pushed_at']
-    return json.dumps(result)
-
-def getESProjectOutput(repo_json):
-    vsResults = runVirusScan(repo_json['cloned_project_path'])
-
-    result = {}
-    result['vscan'] = vsResults
-    result['commits'] = repo_json['numCommits']
-    result['contributors'] = repo_json['contributorsCount']
-    result['contributors_list'] = repo_json['contributors']
-    result['created_at'] = repo_json['created_at']
-    result['forks'] = {'forkedRepos': repo_json['forksJson']}
-    result['full_name'] = repo_json['org_obj']['organization'] + '/' + repo_json['project_name']
-    result['languages'] = repo_json['languagesJson']
-    result['organization'] = repo_json['org_obj']
-    result['origin'] = 'PUBLIC'
-    result['project_description'] = repo_json['description']
-    result['project_name'] = repo_json['project_name']
-    result['rank'] = repo_json['calculatedRank']
-    result['readMe'] = {'content': repo_json['readmeRaw']['content'], 'url': repo_json['readmeRaw']['url']}
-    result['releases'] = []
-    result['repository'] = repo_json['project_name']
-    result['repository_url'] = repo_json['html_url']
-    result['stage_id'] = repo_json['_id']
-    result['stars'] = repo_json['stargazers']
-    result['suggest'] = []
-    result['updated_at'] = repo_json['pushed_at']
-    result['watchers'] = repo_json['numWatchers']
-
-    return json.dumps(result)
-
-def createESInsertString(repo_json, index):
-    idstr = json.loads(repo_json)['stage_id']
-    return '{"index": {"_index": "' + index + '", "_type": "project", "_id": "' + idstr + '"}}'
-
-if __name__ == "__main__":
-    document = ''
-    org_repos = get_org_repos()
-    repos = map_repo_attributes(org_repos)
-    clone_repos(repos)
-    for repo in repos:
-        print('Processing ' + repo['project_name'])
-        execute_sonar(repo)
-        repo_with_metrics = get_sonar_metrics(repo)
-
-        es_code_json = getESCodeOutput(repo_with_metrics)
-        es_project_json = getESProjectOutput(repo)
-
-        document += createESInsertString(es_code_json, 'code') + '\r\n'
-        document += es_code_json + '\r\n'
-        document += createESInsertString(es_project_json, 'projects') + '\r\n'
-        document += es_project_json + '\r\n'
-
-        print(repo['project_name'] + ' processed')
-
-
-        break
-
-    # # # send to ES
-    print('Writing data to ES')
-    header = {'Content-type': 'application/json'}
-    es_post_response = requests.post(os.environ['elasticsearch_api_base_url'] + '/_bulk', data=document, headers=header)
-    print('Data written to ES')
```

Comment:
Let's print the elasticsearch response here in case anything goes wrong.

Reply:
will do